### PR TITLE
Feat/update electron and friends

### LINKED ISSUE #5537: Dependencies(suite-desktop): update deprecated electron
**Info**
We are using `electron v15`, which got deprecated. Update to `electron v18`. 

**Issues**

- it needs to be updated also in `nix.shell` 
- we need to update `node` to version `16`

### PR DESCRIPTION
**Pull request by tomasklim**
Waiting for #5585

Closes #5537

### CODE REVIEW

File: scripts/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "type-check": "tsc --build"
     },
     "dependencies": {
-        "chalk": "^5.0.0",
+        "chalk": "^4.1.2",
```
Comment (dahaca) [inline scripts/package.json:10]: Why this?

**Review by dahaca — COMMENTED** [review]
Why not Electron 19.0.0?

**Comment by tomasklim** [comment]
> Why not Electron 19.0.0?

Nix shell is not yet released with Electron v19. That means that e.g. we would be building app with Electron v19, but Marek would build it with Electron v18. This is also why we need to wait for this #5585 otherwise it would be built with Electron v15

File: scripts/package.json (hunk 1 above)
Comment (tomasklim) [reply scripts/package.json:10]: I am unifying versions across our app.

https://github.com/chalk/chalk/releases/tag/v5.0.0

`If you use TypeScript, you will want to stay on Chalk 4 until TypeScript 4.6 is out. https://github.com/microsoft/TypeScript/issues/46452`

TypeScript 4.6 is out but we are using 4.5.5

**Comment by tomasklim** [comment]
It will fix https://github.com/trezor/trezor-suite/security/dependabot/92

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**